yt_dlp/extractor/roosterteeth.py

```diff
@@ -103,15 +103,15 @@ def _real_extract(self, url):
                 api_episode_url + '/videos', display_id,
                 'Downloading video JSON metadata')['data'][0]
             m3u8_url = video_data['attributes']['url']
-            subtitle_m3u8_url = video_data['links']['download']
+            # XXX: additional URL at video_data['links']['download']
```

Comment:
It gives slightly different data, but the media stream looks like the same. Figured it’s not necessary to process it here, but it might be useful in the future as backup.

Reply:
@pukkandan So we just leave that one out? It could easily be included in the formats list as an alternative download source (preference −1).